src/ansys/fluent/core/launcher/launcher_utils.py

```diff
@@ -282,30 +353,6 @@ def _get_mode(mode: Optional[Union[FluentMode, str, None]] = None):
     return mode
 
 
-def _raise_exception_g_gu_in_windows_os(additional_arguments: str) -> None:
-    """If -g or -gu is passed in Windows OS, the exception should be raised."""
-    additional_arg_list = additional_arguments.split()
-    if _is_windows() and (
-        ("-g" in additional_arg_list) or ("-gu" in additional_arg_list)
-    ):
-        raise InvalidArgument("Unsupported '-g' and '-gu' on windows platform.")
-
-
-def _update_launch_string_wrt_gui_options(
```

Comment:
I have removed this function as user will now use the `exposure` argument instaed of `additional_arguments` for `-g` or `-gu`.